src/main/java/org/jenkinsci/test/acceptance/utils/ElasticTime.java

```diff
@@ -59,7 +59,16 @@ public long seconds(long secs) {
     }
 
     public long milliseconds(long ms) {
-        double actualSeconds = concurrency * factor;
+        double actualSeconds = getSlowDownFactor();
         return Math.round(ms * actualSeconds);
     }
+    
+    /**
+     * Get the factor by which we slow down time.
```

Comment:
_If my calculations are correct, when this baby hits eighty-eight miles per hour… you’re gonna see some serious shit._

**Doc, “Back To The Future”**